### PR TITLE
Feature: Use WebpackMissingModule

### PR DESCRIPTION
**Pull request by EugeneHlushko**
**What kind of change does this PR introduce?**
Use `WebpackMissingModule` instead of multiple strings.

**Did you add tests for your changes?**
No

**If relevant, link to documentation update:**

**Summary**
Following a TODO item

**Does this PR introduce a breaking change?**
no

**Other information**

### CODE REVIEW

File: test/configCases/errors/multi-entry-missing-module/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+it("Should use WebpackMissingModule when module is missing with multiple entry setup", function() {
+  var fs = require("fs");
+  var source = fs.readFileSync("test/js/config/errors/multi-entry-missing-module/b.js", "utf-8");
```
Comment (EugeneHlushko) [inline test/configCases/errors/multi-entry-missing-module/index.js]: @sokra any idea why both 
```
fs.readFileSync("./b.js", "utf-8");
```
and 
```
fs.readFileSync(path.join(__dirname, "b.js"), "utf-8");
```

keep failing with file not found?

Comment (sokra) [reply test/configCases/errors/multi-entry-missing-module/index.js]: You need to add

``` js
node: {
  __dirname: false
}
```

to config. This allows you to use node.js' `__dirname`

**Comment by EugeneHlushko** [comment]
@sokra thank you, forgot about node options

**Comment by sokra** [comment]
Thanks

**Pull request merged**